_maps/map_files/tramstation/tramstation.dmm

```diff
@@ -3255,7 +3260,9 @@
 /obj/effect/turf_decal/siding/thinplating{
 	dir = 8
 	},
-/obj/structure/fluff/tram_rail/floor,
+/obj/structure/fluff/tram_rail/floor{
+	layer = 601
```

Comment:
This value is way too high, and should really not be done in map edits. it'll lead to drift, and annoying to fix failures

Reply:
The crossings are on layer 600, I imagine because of in layers.dm
`//#define TURF_LAYER 2 //For easy recordkeeping; this is a byond define. Most floors (FLOOR_PLANE) and walls (GAME_PLANE) use this.
#define OPENSPACE_LAYER 600 //Openspace layer over all turfs`

Reply:
it still should not be done in var edits, this is why we have defines

Reply:
also, turfs are not on the same plane as these things, at least I hope, so it shouldn't matter

Reply:
They're layer 2 in strongdmm and they're on layer 600 in game and I'm not sure where that's coming from.

Reply:
Current state in game, it'd be nice to fix it that way fixing the layer of the turfs.

Reply:
> They're layer 2 in strongdmm and they're on layer 600 in game and I'm not sure where that's coming from.

it's from the turf transparency element, @GoldenAlpharex was poking at it a bit ago

Reply:
Yeah I was poking at it tonight because layer 600 is dumb. I'm thinking changing it to catwalks, fits in the industrial setting anyways.

Reply:
Oranges said don't touch it.

Reply:
We don't want to fuck up the OPENSPACE_LAYER for everywhere else it's being used.

Reply:
I'm still testing, but I think we can make the layers 2.4 and 2.5 or something, don't need the giant increase.

Reply:
that layer bit does nothing. it was likely done to ensure the turf layers over all the things in its viscontents, but that's not a problem anymore. you should just remove it and things will work as they should

Reply:
Yeah I've been working on it not needing the layer, but leaving a draft until I've fully tested it.

Reply:
Yeah made it a proper define for the layer, now everything is layered properly within layer 2.

Reply:
Would you mind removing the transparent later stuff from the transparency element? it does not need to exist

Reply:
Sure, but I don't quite understand what you want removed?

Reply:
![image](https://user-images.githubusercontent.com/58055496/204085648-19310967-67c9-4d56-9114-01c7c528f1c5.png)
this line
![image](https://user-images.githubusercontent.com/58055496/204085662-c87cf4bb-fa1f-43c7-aa7e-96b866aba5ff.png)
and this one

Reply:
Nice, just tested this. It works and means we can keep using glass for the tram crossings.